tests/test_server_basic.py

```diff
@@ -1525,5 +1525,6 @@ def test_param_value(dispatcher_live_fixture):
     assert c.status_code == 200
     print("content:", c.text)
     jdata=c.json()
-
-    assert jdata['products']['echo']['ang'] == 2.0
\ No newline at end of file
+    # TODO notice the difference, is this acceptable?
+    assert jdata['products']['analysis_parameters']['ang'] == 0.0005555555555555556
```

Comment:
This dictionary is also used when, e.g., generating api_code. But dispatcher expects it in arcsec, not in deg, so the api_code is wrong.
This works for Time, as there is T_format parameter in the request. Introducing units as parameter for every parameter available is definitely overkill.

Reply:
Good point regarding the api_code. A possible solution could be a "formatting" of the dict to be prepared for the generation of the api_code